### DISCUSSION TITLE
Impianto smart BTicino

### DISCUSSION THREAD

**Discussion by alopozzi** [indomus/forum#3329] [Nuove idee, proposte, progetti e guide]
Buonasera a tutti! Sto ristrutturando casa e rifacendo l'impianto elettrico. L'elettricista mi sta installando un impianto tradizionale smart, con gateway e comandi per tapparelle connessi. Gli ho chiesto se era possibile installare i comandi digitali living now della Bticino senza cambiare la tipologia di impianto (non mi interessa la domotica) ma mi ha detto di no....me lo confermate che non c'è nessuna possibilità? Mi piacerebbe installare comandi di questo tipo.....eventualmente conoscete modelli simili anche non della BTicino? Grazie a chiunque vorrà rispondermi

**Comment by TeoLinuX** [comment]
Secondo me si può, basta che nel quadro elettrico inserisca il modulo che fa da "hub" della domotica, che se non sbaglio è il Gateway per smart home BTICINO FC80GT.

Disclaimer: io ho BTicino Living now, ma le uniche cose smart sono il termostato Smarther2 e il videocitofno cx100. Non ho il gateway e le tapparelle.

**Comment by TeoLinuX** [comment]
Aggiungo la mia esperienza. Io ho acquistato casa un anno fa in costruzione e non sapevo nulla di domotica, purtroppo.
A posteriori i miei consigli:
- tirare corrugati ovunque come se non ci fosse un domani, per avere predisposizione per fare passare cavi ethernet o fibra. Prevedere lo spazio per un buon switch, possibilmente POE e possibilmente managed. Magari non lo si acquista subito, ma con il tempo....
- tirare cavi elettrici per portare alimentazione in alcuni punti in alto dei mobili o muri (per mettere sensori o IR blaster etc)
- separare la rete (con diversi SSID o VLAN) fra quella per domotica e quella per i dati dei vari PC e TV, console, smartphone.
Se si vuole fare un hub homeassistant, metterlo in VM su proxmox e usaere il più possibile dispositivi zigbee. Se si vuole restare semplicemente con i cloud dei produttori, scegliere un ecosistema (aqara, sonoff, tuya, ...) e comprare il loro hub fisico.

**Reply by misterbit74** [reply]
Concordo con i primi due punti al 100% ...  parole sante
Per il terzo non sono un fan del cloud e non mi piace riempire casa di onde radio.
Ho riscontrato una certa lentezza e se poi internet e' giu' non riesci a comandare piu' niente.

Fai tutto in locale via cavo e usa HomeAssistant come cuore del tuo sistema.

**Reply by TeoLinuX** [reply]
Sì, anche io cablo tutto quello che posso (e non  a caso ho suggerito di mettere corrugati e predisposizioni sopra i mobili ovunque possibile). Anche io non sono fan del wifi, ma zigbee è molto meglio in termini di irraggiamento di onde radio, inoltre alcuni sensori o dispositivi ci sono solo wireless. E tra i wireless, in termini di qualità, controllo e stabilità, zigbee batte il wifi per la domotica. 
Sul cloud, d'accordissimo. Se uno si fa un homeassitant lo usa come hub e riesce a svincolarsi dal 99% dei cloud, soprattutto se usa una antenna coordinator zigbee, meglio an!  
Parlavo di ecosistemi solo per chi non vuole installare e configurare HA, ma vuole una soluzione chiavi in mano.

**Reply by misterbit74** [reply]
Ai tempi mi ero fatto fare un paio di preventivi con una soluzione chiavi in mano della bticino myhome...
Mamma mia ho ancora gli incubi alla notte... il preventivo era stratosferico !!!

Felicissimo di aver scelto HomeAssistant.

**Comment by snisko** [comment]
Ciao,
io ho BTicino Living Now e ti confermo che per ogni comando (tapparella o luce che sia), così come per le prese di corrente UNEL (intendo quelle trivalenti "Shuko+10A+16A") esiste sia la versione "connessa" che la versione "non connessa".
Mi pare di capire dal tuo post che tu sia interessato alla linea estetica della Living Now, ma non ti interessi per niente la versione "smart". Bene, lo puoi fare!

Ma se nel mentre tu avessi cambiato idea, ti aggiungo che io ho montato tutti i comandi (non le prese) in versione connessa e sono riuscito a domotizzare tutto senza l'utilizzo del Gateway, grazie a Home Assistant, un coordinator Sonoff ZBDongle-E ed il Bridge-Gateway Zigbee2MQTT. Quindi senza passare dal cloud di BTicino! Ho solo difficoltà, ma ci voglio arrivare, ad integrare il videocitofono (EOS300 with Netatmo) senza passare dal Cloud di BTicino.

**Reply by Gh0s7** [reply]
@snisko ti dispiacerebbe aggiungere qualche dettaglio? Stavo valutando una soluzione simile (quindi "estetica" BTicino ma gestione via HA)

**Reply by snisko** [reply]
Di che dettagli hai bisogno?
Io ho cablato casa come un impianto "tradizionale" e ho montato relè (3584C) oppure deviatori connessi (K4003C) per il comando delle luci, ho montato comandi tapparelle connessi (K4027C).
Poiché tali comandi comunicano via protocollo ZigBee, hai bisogno di un coordinator (io uso un Sonoff ZBDongle-E collegato al mio Raspberry) e di un Bridge-Gateway installato su HA (io uso Zigbee2MQTT).
Quando hai fatto questo devi sono far partire l'abbinamento tra bridge-gateway e modulo connesso.
Ovvero io abilito "permetti Joint:* su Z2M, e premo sul comando connesso il pulsante di reset in modo che vada in pairing mode. Z2M dopo poco trova il comando e da lì lo gestisci via HA.

Più dettagli di così non me ne vengono in mente. Se ti serve altro chiedi pure.
Saluti

**Reply by matthewdisa** [reply]
Ciao! Interessato alla discussione, anche io ho un impianto cablato ''tradizionale'', con bticino Matix, volevo domotizzarlo e utilizzare HA come gestionale. Allora le tapparelle sono motorizzate, wifi e in alcune stanze anche cavi ETH, (circa 85mq) e il tutto arriva in un armadietto dove ho Router, switch e NAS. Userei o quest'ultimo (ma è un ''vecchio Synology 216play non credo sia il massimo)  o un mini PC con caratteristiche adeguate. Vorrei poter inserire in HA la gestione tapparelle, le luci, non per forza dimmerabili, i condizionatori, la caldaia e quindi i termosifoni, magari qualche presa per gestire i carichi e in futuro magari allarme ed altre cosine simpatiche. Per iniziare di cosa avrei bisogno? zigbee, matter, shelly...non so da dove cominciare e con cosa...Grazie a tutti
